src/Svelte/assets/dist/render_controller.js

```diff
@@ -1,11 +1,18 @@
 import { Controller } from '@hotwired/stimulus';
 
 class default_1 extends Controller {
+    app;
```

Comment:
Do you know why only the controller have changed and not other .js files (example in Svelte folder) ?